.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -0,0 +1,128 @@
+# How to use this GitHub Actions workflow
```

Comment:
I think this should be a file available somewhere in the repos or the documentation that you copy to your own github workflows. Otherwise there won't be any visibility, and it's like it's a workflow for the current repos.

Reply:
I still disagree with having this file in the .github folder by default. It should only be in some samples/docs folder, and explained for devs to customize it and put it there if they want it. Most users don't use GH as their CI for Orchard. It a very custom flow that is nice but you are part of a few doing it. If you make the file available and some doc on how to use it, then it would be nice to add a pointer to it in the README for sure.

Reply:
I agree that it shouldn't be in this repo directly (we have CI workflows for this repo since then and it would appear there), so I'll close this PR. Please open a new one if you'd like to add this to the docs.
Also check out https://github.com/Lombiq/GitHub-Actions/blob/dev/.github/workflows/deploy-orchard1-to-azure-app-service.yml, we use this for our own O1 applications, but you can fully customize it through parameters.